lib/ansible/modules/cloud/docker/docker_container.py

```diff
@@ -1510,7 +1510,7 @@ def default_host_ip(self):
                         break
                 except NotFound as dummy:
                     self.client.fail(
-                        "Cannot inspect the network '{0}' to determine the default IP: {1}".format(net['name'], e),
+                        "Cannot inspect the network '{0}' to determine the default IP: {1}".format(net['name'], ip),
```

Comment:
remove 'as dummy' as it is unused

Reply:
I see what this was meant to do now: display the exception message. The variable name was changed in the exception, just not the format statement.